leonardnjura/wanderlust
# Setup algolia search

[#182760314](https://www.pivotaltracker.com/story/show/182760314)

### What does this PR do?
- adds algolia search [ai]
- adds alternate local search [simple regex]

#### How should this be manually tested?
- setup next.js project, 
- run `$ git clone <repo>` and `$ cd <branch>`
- run `$ yarn install` 
- run `$ yarn dev`
- invoke search: tap icon on frontend header; type and observe; 
- click the algolia result item, or hit enter to continue with query term into the simple regex search results page

#### Screenshots
